### PR TITLE
Add script to patch the console config

### PR DESCRIPTION
**Pull request by vbnrh**
Signed-off-by: Vineet Badrinath <vineetbnath@gmail.com>

### CODE REVIEW

File: test-prow-e2e.sh [hunk 1]
```diff
@@ -9,6 +9,18 @@ if [ $# -eq 0 ]
...
+export CONSOLE_CONFIG_NAME="cluster"
+export ODF_PLUGIN_NAME="odf-console"
+
+echo "Enabling Console Plugin for ODF Operator"
+oc patch console.v1.operator.openshift.io ${CONSOLE_CONFIG_NAME} --type=json -p="[{'op': 'add', 'path': '/spec/plugins/-', 'value':\"${ODF_PLUGIN_NAME}\"}]"
+
+# Wait for 2 mins to enable the console plugin
+echo "Waiting for odf-console plugin to be enabled"
+sleep 120
+echo "Console plugin enabled"
```
Comment (bipuladh) [inline test-prow-e2e.sh:22]:
```suggestion
```

**Review by bipuladh — COMMENTED** [review]
You don't need to wait for console plugin to get enabled. It only takes time for the console pod to come up or for the operator to create console plugin CR.

File: test-prow-e2e.sh (hunk 1 above)
Comment (vbnrh) [reply test-prow-e2e.sh]: ack

**Review by bipuladh — APPROVED** [review]
/lgtm

**Pull request merged**